ecolex/management/config/subjects.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Are these files (json & xml) supposed to be included in the repo? We can obtain them through the management commands anyway. Maybe we should add steps in deploy for downloading / processing these files.

Reply:
It's ok, tracking change history on those dictionaries can be handy